lib/widgets/chat.dart

```diff
@@ -103,110 +106,141 @@ class _ChatMessageState extends State<ChatMessage> {
+                              gradient: widget.props.primaryGradient,
+                              shape: BoxShape.circle,
+                            ),
+                            child: CircleAvatar(
+                              radius: 16,
+                              backgroundColor: Colors.transparent,
+                              backgroundImage:
+                                  (msg.user.profilePhotoUrl != null)
+                                      ? NetworkImage(msg.user.profilePhotoUrl)
+                                      : null,
+                              child: (msg.user.profilePhotoUrl != null)
+                                  ? null
+                                  : (msg.user != null &&
+                                          msg.user.fullName == null)
+                                      ? Text(
+                                          msg.user.email
+                                              .substring(0, 1)
+                                              .toUpperCase(),
+                                          style: TextStyle(
+                                              color:
+                                                  Theme.of(context).cardColor),
+                                        )
+                                      : Text(
+                                          msg.user.fullName
+                                              .substring(0, 1)
+                                              .toUpperCase(),
+                                          style: TextStyle(
+                                              color:
+                                                  Theme.of(context).cardColor),
+                                        ),
+                            ),
+                          )
+                        : SizedBox(
+                            width: 32,
+                          ),
+                  ),
+                if (userSent)
+                  TimeWidget(
+                    userSent: userSent,
+                    msg: msg,
+                    isVisible: isTimeSentVisible,
+                  ),
+                Container(
+                  decoration: BoxDecoration(
+                    color: userSent
+                        ? widget.props.primaryColor
+                        : Theme.of(context).brightness == Brightness.light
+                            ? brighten(Theme.of(context).disabledColor, 80)
+                            : Color(0xff282828),
+                    gradient: userSent ? widget.props.primaryGradient : null,
+                    borderRadius: BorderRadius.circular(4),
+                  ),
+                  constraints: BoxConstraints(
+                    maxWidth: maxWidth,
+                  ),
+                  margin: EdgeInsets.only(
                     top: (isFirst) ? 15 : 4,
-                    bottom: 5,
+                    bottom: 4,
+                    right: userSent ? 18 : 0,
```

Comment:
@aguilaair I removed the margin: left here, is it safe?

Reply:
Yep, shouldn't matter if it still looks right.

Reply:
Actually, doesn't quite work:
![imagen](https://user-images.githubusercontent.com/29983481/102719679-9c30e880-42ef-11eb-8a1b-e861adaa343a.png)

Reply:
Forget it, in your PR it looks fine